crates/swc_css_codegen/tests/fixture/selectors/class/output.min.css

```diff
@@ -1 +1 @@
-.\10FFF abc:after{content:"\\10FFF abc"}.\10FFFFabc:after{content:"\\10FFFFabc"}.\10FFFF abc:after{content:"\\10FFFF abc"}.\FEFF:after{content:"uFEFF"}
+.𐿿abc:after{content:"\\10FFF abc"}.􏿿abc:after{content:"\\10FFFFabc"}.􏿿abc:after{content:"\\10FFFF abc"}.﻿:after{content:"uFEFF"}
```

Comment:
Expected, because `\10FFF abc` == `𐿿abc` (you can test it manullay in chrome):

```html
<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
    <style>

        .\10FFF abc{
            background: red
        }

    </style>
</head>
<body>
<ul><li>test</li> <li>test</li></ul>
<div class="𐿿abc">test</div>
</body>
</html>
```